MAINTAINERS.md

```diff
@@ -0,0 +1,15 @@
+This page lists all active maintainers of this repository. If you were a
+maintainer and would like to add your name to the Emeritus list, please send us a
+PR.
+
+See [CONTRIBUTING.md](https://github.com/grpc/grpc-community/blob/master/CONTRIBUTING.md)
+for general contribution guidelines.
+
+## Maintainers (in alphabetical order)
+- [mit-mit](https://github.com/mit-mit), Google Inc.
+- [nichite](https://github.com/nichite), Google Inc.
+- [szakarias](https://github.com/szakarias), Google Inc.
+- [sigurdm](https://github.com/sigurdm), Google Inc.
```

Comment:
@jonasfj @domesticmouse and @mkustermann each might or might not belong here.

Reply:
I don't think I'm an active maintainer.

Reply:
I think this is actually a list of people with write-bits enabled.

Reply:
I'm not sure I see much reason to expand it beyond those that consider themselves maintainers